app/population/controllers/daily.js

```diff
@@ -0,0 +1,17 @@
+const populationToGrid = require('../../../common/presenters/population-to-grid')
+const transfersToGrid = require('../../../common/presenters/transfers-to-grid')
+
+function daily(req, res) {
+  const { population } = req
+  const transfers = transfersToGrid()
+
```

Comment:
Any reason for this linebreak? I thought eslint should catch things like this...

Reply:
I think this line is eviewing old code, but I've removed a linebreak anyway.